versioned_docs/version-1.0/language/access-control.md

```diff
@@ -322,6 +323,13 @@ refEF.a
 refEF.b
 // valid, because `refEF` has both of the two required entitlements
 refEF.c
+
+// invalid, because `refEOrF` might not have an `E` entitlement (it may have `F` instead)
+refEOrF.a
+// valid, because `refEOrF` has one of the two entitlements necessary
+refEOrF.b
+// invalid, because `refEOrF` is only known to have one of the two required entitlements
+refEOrF.c
 ```
```

Comment:
Could you please also add a valid example for when a disjunction is wanted, and a disjunction is provided, like e.g. discussed in onflow/cadence#3145?

Reply:
I think we already have this in https://github.com/onflow/cadence-lang.org/blob/45f177ac9de557ed67e81ecb3f0f73c381f76297/versioned_docs/version-1.0/language/references.mdx